javascript/app.js

```diff
@@ -52,7 +65,7 @@ FinancialFreedom.controller('HeaderController', ['$scope', '$location',  functio
     };
 
     $scope.tabsAreVisible = function() {
-        if ($location.path() == '/about' || $location.path() == '/time-to-retirement') {
+        if ($location.path() == '/about' || $location.path() == '/time-to-retirement' || $location.path() == '/') {

```

Comment:
Yeah good call, I knew it was a short term hack. Check out the way I've done it in the next version.